### PR TITLE
[Feature][doc] Add V2 connector jdbc document parameter can speed up data import PR

### PR DESCRIPTION
**Pull request by shuihua777**
https://github.com/apache/seatunnel/issues/6167

### Check list

* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [x] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [x] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/sink/Jdbc.md [hunk 1]
```diff
@@ -186,10 +187,14 @@ Option introduction：
...
 `ERROR_WHEN_DATA_EXISTS`：When there is data, an error is reported
 
-### custom_sql[String]
+### custom_sql [String]
 
 When data_save_mode selects CUSTOM_PROCESSING, you should fill in the CUSTOM_SQL parameter. This parameter usually fills in a SQL that can be executed. SQL will be executed before synchronization tasks.
 
+### enable_upsert [boolean]
+
+Enable upsert by primary_keys exist, If the task only has `insert`, setting this parameter to `false` can speed up data import
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/Jdbc.md]: PTAL @hailin0

File: docs/en/connector-v2/sink/Mysql.md [hunk 2]
```diff
@@ -80,10 +80,11 @@ semantics (using XA transaction guarantee).
...
+| data_save_mode                            | Enum    | No       | APPEND_DATA                  | Before the synchronous task is turned on, different processing schemes are selected for data existing data on the target side.                                                                                                                 |
+| custom_sql                                | String  | No       | -                            | When data_save_mode selects CUSTOM_PROCESSING, you should fill in the CUSTOM_SQL parameter. This parameter usually fills in a SQL that can be executed. SQL will be executed before synchronization tasks.                                     |
+| enable_upsert                             | Boolean | No       | true                         |                                                                                                                                                                                                                                                |
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/Mysql.md]: miss description.

File: docs/en/connector-v2/sink/Oracle.md [hunk 3]
```diff
@@ -74,10 +74,11 @@ semantics (using XA transaction guarantee).
...
+| data_save_mode                            | Enum    | No       | APPEND_DATA                  | Before the synchronous task is turned on, different processing schemes are selected for data existing data on the target side.                                                                                                                 |
+| custom_sql                                | String  | No       | -                            | When data_save_mode selects CUSTOM_PROCESSING, you should fill in the CUSTOM_SQL parameter. This parameter usually fills in a SQL that can be executed. SQL will be executed before synchronization tasks.                                     |
+| enable_upsert                             | Boolean | No       | true                         |                                                                                                                                                                                                                                                |
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/Oracle.md]: ditto

File: docs/en/connector-v2/sink/Mysql.md (hunk 2 above)
Comment (shuihua777) [reply docs/en/connector-v2/sink/Mysql.md]: Add later

File: docs/en/connector-v2/sink/Oracle.md (hunk 3 above)
Comment (shuihua777) [reply docs/en/connector-v2/sink/Oracle.md]: @Hisoka-X I just updated the description

**Comment by shuihua777** [comment]
The latest PR add : OceanBase、Kingbase、Vertica、Snowflake、SqlServer、PostgreSql jdbc parameter an related information

**Comment by Hisoka-X** [comment]
The code style check failed. Please run `mvn spotless:apply` to fix it
image

**Comment by shuihua777** [comment]
I have completed formatting

**Comment by Hisoka-X** [comment]
Oops, there are some conflict should be resolved.

**Comment by shuihua777** [comment]
ok

File: docs/en/connector-v2/sink/Jdbc.md (hunk 1 above)
Comment (hailin0) [inline docs/en/connector-v2/sink/Jdbc.md]:
```suggestion
Enable upsert by primary_keys exist, If the task has no key duplicate data, setting this parameter to `false` can speed up data import
```

Comment (shuihua777) [reply docs/en/connector-v2/sink/Jdbc.md]: other engine i change the description later

**Comment by shuihua777** [comment]
@hailin0 I have completed modifying the descriptions of jdbc engines

**Review by Hisoka-X — APPROVED** [review]

**Review by zhilinli123 — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**